src/biogeophys/SoilStateInitTimeConstMod.F90

```diff
@@ -371,20 +372,26 @@ subroutine SoilStateInitTimeConst(bounds, soilstate_inst, nlfilename)
                 sand = sand3d(g,1)
                 om_frac = organic3d(g,1)/organic_max
              else if (lev <= nlevsoi) then
+                found = 0  ! reset value
+                ! Search within the dataset's range of zisoifl values
                 do j = 1,nlevsoifl-1
-                   ! NOTE(wjs, 2019-08-01) It appears that the code currently doesn't set
-                   ! clay, sand and om_frac explicitly under some conditions. It probably
-                   ! should. My understanding is that currently things work okay, though,
-                   ! because clay, sand and om_frac will remain set at their previous
-                   ! values, which is probably reasonable enough. See also
-                   ! <https://github.com/ESCOMP/ctsm/pull/771#discussion_r309509596>.
```

Comment:
Temporarily reverted the next line to the way it was before this PR, in order to expect bfb from all tests.